tools/test/executable_benchmarks.mlir

```diff
@@ -0,0 +1,19 @@
+// RUN: iree-compile %s -o ignored.mlir \
+// RUN:     --iree-hal-target-backends=vmvx \
+// RUN:     --iree-hal-dump-executable-benchmarks-to=- | \
+// RUN: iree-compile - | \
+// RUN: iree-benchmark-module | \
+// RUN: FileCheck %s
+
+// This test relies on us piping stdout and that there's only a single
+// executable (otherwise we'd need to look at files and that's harder
+// cross-platform). Real automation of this requires xargs: compile and dump a
+// directory of .mlir benchmarks, xargs them to iree-compile to produce the
+// vmfbs (optionally with different flags for each), then xargs to
+// iree-benchmark-module to run them.
+
+// CHECK: BM_abs_dispatch_0_vmvx_bytecode_fb_abs_dispatch_0_generic
+func.func @abs(%input : tensor<f32>) -> (tensor<f32>) {
+  %result = math.abs %input : tensor<f32>
```

Comment:
https://github.com/iree-org/iree/runs/7861809750?check_suite_focus=true

```
/home/runner/actions-runner/_work/iree/iree/tools/test/executable_benchmarks.mlir:17:13: error: custom op 'math.abs' is unknown
  %result = math.abs %input : tensor<f32>
            ^
```
wat?

oh, this should be `math.absf` now

Reply:
ahh